tlatools/src/tla2sany/modanalyzer/SpecObj.java

```diff
@@ -987,4 +990,4 @@ public void setGlobalContextErrors(Errors globalContextErrors)
         this.globalContextErrors = globalContextErrors;
     }
 
-}
```

Comment:
I think this line is just saying that there was no line break after this closing brace previously and my editor added it automatically. I'm trying to keep a clean diff on this PR, but I'm inclined not to revert this change. Let me know if you disagree.